resources/assets/js/core/components/powerToggle.vue

```diff
@@ -61,9 +65,11 @@ export default {
         currentTitle() {
             if (this.active) {
                 return this.titleOn || this.title;
+            } else if (this.disabled) {
+                return this.titleDisabled || this.title;
+            } else {
+                return this.titleOff || this.title;
```

Comment:
I wouldn't add an extra prop just for one use case here. I don't think the separate props for on/off titles were a good idea in the first place. We can just modify the title prop dynamically. You can either do this here, too, or use two conditional components like this:

```html
<power-toggle v-if="labelBotEnabled" title="My enabled title"></power-toggle>
<power-toggle v-else :disabled="true" title="My disabled title"></power-toggle>
```